.travis.yml

```diff
@@ -1,35 +1,51 @@
+language: python
+
 notifications:
   email: false
 
-language: python
-python:
-    - "2.7"
+matrix:
+  include:
+    - os: linux
+      python: 2.7
+      env: MINICONDA_NAME=Miniconda2-latest-Linux-x86_64.sh
+      sudo: required
+    - os: osx
+      language: generic
```

Comment:
is there a reason that language here is `generic`?

Reply:
Yes. The OSX images that are provided by Travis are much less diverse. And there is no pre-installed python image available.

The reason that we first define python as a language for all jobs and then reset this for OSX is to account for weird (and probably buggy) behavior of the Travis scheduler which falls back to using ruby for everything of there no global language defined.